apps/src/code-studio/components/SoundPicker.jsx

```diff
@@ -103,20 +105,31 @@ export default class SoundPicker extends React.Component {
         soundPlayer={this.props.soundPlayer}
       />
     ) : (
-      <SoundLibrary assetChosen={this.getAssetNameWithPrefix} />
+      library
     );
 
-    return (
-      <div className="modal-content" style={styles.root}>
-        {title}
-        {this.props.showUnderageWarning && (
-          <p style={styles.warning}>
-            Warning: Do not upload anything that contains personal information.
-          </p>
-        )}
-        {modeSwitch}
-        {body}
-      </div>
-    );
+    if (this.props.libraryOnly) {
```

Comment:
If we change the conditional on line 97 in the ternary that sets the body to also check whether we are in `libraryOnly` mode and use that to determine the body, we could avoid duplicating some of this code. (Might need to adjust the conditional for displaying the showUnderageWarning as well?)